docs/specs/error.yml

```diff
@@ -1,28 +1,30 @@
----
-# Exceed max file error: Suggestions
+﻿# Exceed max file error: Suggestions
 noDryRun: true
 inputs:
   docfx.yml: |
     markdownValidationRules: rules.json
     MaxFileSuggestions: 2
-  rules.json: |
+  rules.json: |-
     {
-      "images": {
-        "rules": [
-          {
-            "type": "ImageAltTextEqualsFilename",
-            "message": "AltText {0} is the same as the image file name.",
-            "code": "altText-equal-filename",
-            "severity": "SUGGESTION",
-            "exclusions": [
-              "includes",
-              "landingpage",
-              "hubpage",
-              "toc"
+        "images": {
+            "rules": [
+                {
+                    "type": "ImageAltTextEqualsFilename",
+                    "message": "AltText {0} is the same as the image file name.",
+                    "code": "altText-equal-filename",
+                    "severity": "SUGGESTION",
+                    "contentTypes": [
```

Comment:
What is the behavior if there is no `contentTypes`? It seems a lot of burden to force each rule to have this list of `contentTypes`. If there is a default set, we could remove this property for most of the tests using regex. The python script alters the formatting of YAML files (which may conflict with manual edits).

Reply:
OK